### PR TITLE
cmd set_inv_iq added

### PR DESCRIPTION
**Pull request by REGIOIGER**
Added command to invert IQ signal (useful to sniffing downlink packets)

### CODE REVIEW

**Review by DeimosHall — COMMENTED** [review]
I don't know what it does, but looks good for me

File: Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino [hunk 1]
```diff
@@ -136,7 +139,7 @@ void set_freq(){
   arg = SCmd.next();
   frequency = atof(arg);
   if (arg != NULL){
-    if(frequency > 902 && frequency < 923){
```
Comment (wero1414) [inline Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino:139]: This 923 frequency only applies for US915, but any user should be able to put any frequency they desire

File: Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino [hunk 2]
```diff
@@ -485,6 +493,35 @@ void set_chann(){
   }
 }
 
+void set_inv_iq(){
```
Comment (wero1414) [inline Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino:496]: good implementation, no problem, maybe i would add the use case in the help menu

**Review by wero1414 — CHANGES_REQUESTED** [review]
Check range of frequencies that this firmware will be supporting

File: Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino (hunk 1 above)
Comment (REGIOIGER) [reply Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino:139]: Frequency restriction removed. TODO: restrict frequency and channels according to region

File: Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino [hunk 3]
```diff
@@ -139,16 +141,16 @@ void set_freq(){
   arg = SCmd.next();
   frequency = atof(arg);
   if (arg != NULL){
-    if(frequency > 902 && frequency < 928){
+//    if(frequency > 902 && frequency < 928){
```
Comment (DeimosHall) [inline Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino:144]: If It isn't necessary, it could be deleted

**Review by DeimosHall — COMMENTED** [review]
Remove commented code

**Comment by sabas1080** [comment]
@REGIOIGER status??

File: Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino (hunk 3 above)
Comment (REGIOIGER) [reply Fw/LoRa_PHY/LoRaSniffer/LoRaSniffer.ino]: the change was made

**Comment by REGIOIGER** [comment]
Ready to merge

**Review by sabas1080 — APPROVED** [review]

**Pull request merged**